main/libxmlsec/xmlsec1-configure.patch

```diff
@@ -759,7 +759,7 @@
  XMLSEC_OPENSSL_ALIBS    = libeay32.lib wsock32.lib kernel32.lib user32.lib gdi32.lib
  
 -XMLSEC_NSS_SOLIBS   	= smime3.lib ssl3.lib nss3.lib libnspr4.lib libplds4.lib libplc4.lib kernel32.lib user32.lib gdi32.lib
-+XMLSEC_NSS_SOLIBS   	= smime3.lib nss3.lib nspr4.lib kernel32.lib user32.lib gdi32.lib
++XMLSEC_NSS_SOLIBS   	= smime3.lib nss3.lib licnspr4.lib libplds4.lib libplc4.lib kernel32.lib user32.lib gdi32.lib
```

Comment:
This li*c*nspr4.lib looks like a typo...?

Reply:
That's definitely a typo, but it did build anyway... ;-)

Reply:
Fixed!